### PR TITLE
Fix/minor 3d fixes

### PR DESCRIPTION
**Pull request by benoitdm-oslandia**
Multiple 3D fixes and enhancements:

* geometry renderer identification
* log messages
* segfault fix
* tessalator bug fix
* improve objectname for 3D entities

### CODE REVIEW

**Comment by benoitdm-oslandia** [comment]
@uclaros, @lbartoletti or @ptitjano can you have a look please?

File: src/3d/symbols/qgspolygon3dsymbol.cpp [hunk 1]
```diff
@@ -170,13 +170,32 @@ void QgsPolygon3DSymbol::setMaterialSettings( QgsAbstractMaterialSettings *mater
...
-  for ( Qt3DRender::QGeometryRenderer *r : renderers )
+  const QList<Qt3DCore::QEntity *> entities = entity->findChildren<Qt3DCore::QEntity *>( QString(), Qt::FindDirectChildrenOnly );
+  if ( entities.isEmpty() )
   {
-    Qgs3DExportObject *object = exporter->processGeometryRenderer( r, objectNamePrefix );
-    if ( object == nullptr ) continue;
-    exporter->processEntityMaterial( entity, object );
-    exporter->mObjects.push_back( object );
+    const QList<Qt3DRender::QGeometryRenderer *> renderers = entity->findChildren<Qt3DRender::QGeometryRenderer *>();
+    for ( Qt3DRender::QGeometryRenderer *r : renderers )
```
Comment (ptitjano) [inline src/3d/symbols/qgspolygon3dsymbol.cpp:177]: nitpick: `renderer` instead of `r`

**Comment by benoitdm-oslandia** [comment]
@uclaros can you have a look please?

**Review by ptitjano — APPROVED** [review]

File: src/3d/symbols/qgspolygon3dsymbol.cpp (hunk 1 above)
Comment (lbartoletti) [inline src/3d/symbols/qgspolygon3dsymbol.cpp:177]:
```suggestion
    for ( Qt3DRender::QGeometryRenderer *renderer : renderers )
```

File: src/3d/symbols/qgspolygon3dsymbol.cpp [hunk 2]
```diff
@@ -170,13 +170,32 @@ void QgsPolygon3DSymbol::setMaterialSettings( QgsAbstractMaterialSettings *mater
...
+    {
+      Qgs3DExportObject *object = exporter->processGeometryRenderer( r, objectNamePrefix );
```
Comment (lbartoletti) [inline src/3d/symbols/qgspolygon3dsymbol.cpp:179]:
```suggestion
      Qgs3DExportObject *object = exporter->processGeometryRenderer( renderer, objectNamePrefix );
```

**Review by lbartoletti — APPROVED** [review]

**Pull request merged**